notebooks/en/agent_change_llm.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<p>"Hugging Face Serverless API and dedicated Endpoints"</p><p> </p><p>I think it'd be helpful to add some links to the Serverless API and Endpoint docs.</p><p> </p><p>"The Serverless API provides a built-in <code>HfEngine</code> class that lets you use any model on the Hub via the Serverless API or your own dedicated Endpoint."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/140/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>also if there's any docs to where someone can create their endpoint would be nice</p>